exporter/zipkin/lib/opentelemetry/exporter/zipkin/exporter.rb

```diff
@@ -148,37 +148,38 @@ def send_spans(zipkin_spans, timeout: nil) # rubocop:disable Metrics/AbcSize, Me
             @http.start unless @http.started?
 
             response = @http.request(request)
-            response.body # Read and discard body
```

Comment:
since we've moved the debug log into the non `SUCCESS` blocks and removed the `body` read from those calls, we should still keep this iiuc? other than that this lgtm